### DISCUSSION TITLE
[Bug]: manifest-beta.json has old version

### DISCUSSION THREAD

**Discussion by erohtar** [RafaelGB/obsidian-db-folder#736] [Q&A] [answered]
### What happened?

I'm using DB Folder with [Obsidian BRAT](https://github.com/TfTHacker/obsidian42-brat) plugin and it allows me to use latest beta before being approved by Obsidian team. 
I believe it reads `manifest-beta.json` which still has an older ver of DB Folder mentioned in it, thus it doesn't update to latest - could you update that file (and keep updating) to the latest release please?
Thank you!

### What platform were you using?

Desktop

### Version of the plugin

v3.0.1

### Are you using the latest version of the plugin in your Obsidian vault?

- [X] I have verified that I am on the latest version

### Are you check if there is a similar issue?

- [X] I have verified that there are not similar issues

**Comment by RafaelGB** [comment, marked as answer]
Hello @erohtar,

This version is the last one compatible with mobiles.  Until Obsidian does not upload its API to 1.1 on mobile I would like to keep said version in manifest-bera to give the user the option to use BRAT

You can always use the frozen version option to install the lastest one.

Sorry for the inconvenience

**Reply by erohtar** [reply]
I see - yes, that makes sense. I'll do as you suggested then.
